nvflare/edge/aggregators/edge_dict_accumulator.py

```diff
@@ -62,4 +67,12 @@ def reset(self, fl_ctx: FLContext):
         self.num_devices = 0
 
     def aggregate(self, fl_ctx: FLContext) -> Shareable:
-        return Shareable({MsgKey.RESULT: self.weights, MsgKey.NUM_DEVICES: self.num_devices})
+        return Shareable(
+            {
+                MsgKey.RESULT: {MsgKey.WEIGHTS: self.weights, MsgKey.MODE: self.mode},
+                MsgKey.NUM_DEVICES: self.num_devices,
+            }
+        )
```

Comment:
we need to clearly define and document the structure we expect somewhere in our documentation